### ISSUE TITLE
Facebook Pixel View Content and Purchase Events stopped loading

### ISSUE THREAD

**Issue by timlumsden** [facebookincubator/facebook-for-woocommerce#390]
Hi guys,

Hoping you can help.  The plugin has been working perfectly over the last 3 months and without warning the View Content and Purchase events stopped loading last week.

This is the exact error message:

"The Facebook pixel code on this page didn't load, so no information was sent to Facebook"

You can take a look at the shop page where the view content event is supposed to fire:  https://www.tradiesgear.com/shop/tradies-belt/

I updated the plugin to the newest version 1.8.3 but the error remains.

Are you able to help, please?

Cheers,

Tim
--

--

**Comment by sammul** [comment]
@timlumsden Did you install some other plug-in around March 29th?

**Comment by dmitridr** [comment]
@timlumsden It seems you have the Google Enhanced E-Commerce plugin installed? This plugin is currently incompatible with ours. 

You can see there's some detailed discussion about this in #319 #329 
One solution is to disable that plugin, though some users have got it working by changing a setting in Google.

**Comment by dmitridr** [comment]
In any case, it would be helpful for us if you try and (temporarily) disable that plugin and check the helper tool again to see if it is the cause or not.

**Comment by dmitridr** [comment]
I looked at the source code for the link you provided. 
There's code I believe is added by the Enhanced-Ecomm extension that adds a `gtag` next to the Facebook ViewContent event that has a syntax error. I would encourage you to report that to the owners of the Enhanced-Ecommerce plugin. The syntax error is there is no semicolon at the end.

```
gtag("event", "view_item", {
--
  | "event_category":"Enhanced-Ecommerce",
  | "event_label":"product_impression_pp",
  | "items": [
  | {
  | "id": tvc_po.tvc_i,// Product details are provided in an impressionFieldObject.
  | "name":  tvc_po.tvc_n,
  | "category":tvc_po.tvc_c,
  | }
  | ],
  | "non_interaction": true
  | })


```

Hope that helps.

**Comment by mengyingdu** [comment]
This was fixed in v1.8.7. Please upgrade to [v1.9.0](https://github.com/facebookincubator/facebook-for-woocommerce/releases/tag/v1.9.0)
